### ISSUE TITLE
importing FigoSession has changed

### ISSUE THREAD

**Issue by berend** [figo-connect/python-figo#48]
Since introducing `pbr` (?) we changed the importing behavior:

old:
```
from figo import FigoSession
```

now:
```
from figo.figo import FigoSession
```

- this is not elegant
- upgrading to this version breaks code

**Comment by holger-zeit** [comment]
Hi, whats the status on #49 ?
Thanks :)

**Comment by berend** [comment]
vacations ... 

I just merged and created a new release, 1.7.3 is now live at https://pypi.python.org/pypi/python-figo.
